src/main/scala/fundamentals/level03/TryExercises.scala

```diff
@@ -86,6 +86,20 @@ object TryExercises {
       case Failure(throwable) => ???
     }
 
+  /**
+    * Write a function that converts a `Try[A]` to `Option[A]`.
+    *
+    * scala> tryToOption(parseIntSafe("1"))
+    * = Some(1)
+    * scala> tryToOption(parseIntSafe("abc"))
+    * = None
+    */
+  def tryToOption[A](tryA: Try[A]): Option[A] =
```

Comment:
They would've learned `.toOption` by this point. Perhaps remove the pattern match so they can solve it however they want.

Reply:
I couldn't find any mention of `.toOption` in the exercises. But if it's in the slides, then I'll remove the exercise. Also the exercise above is `tryToEither`. Is that also redundant?

Reply:
Oops, in `TryExercises`, they would've done `Try => Either`, not to `Option`. We may have also communicated that there's a `.toOption` along with `.toEither` on `Try`.

On second thought I think your addition is fine.